### PR TITLE
pamixer: 1.4 -> unstable-2021-03-29

### PR DESCRIPTION
**Pull request by thiagokokada**
###### Motivation for this change

Bump `pamixer` to latest commit on git.

This brings some bugfixes (like a [memory leak](https://github.com/cdemoulins/pamixer/commit/607df34a630f2485dee03cb3cee7552b84025566)) and also some new features (like `--get-default-sink`, `--set-limit`).

Also add missing manpage.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/pamixer/default.nix [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
   pname = "pamixer";
-  version = "1.4";
+  version = "unstable-2021-03-29";
 
   src = fetchFromGitHub {
     owner = "cdemoulins";
     repo = "pamixer";
-    rev = version;
-    sha256 = "1i14550n8paijwwnhksv5izgfqm3s5q2773bdfp6vyqybkll55f7";
+    rev = "4ea2594cb8c605dccd00a381ba19680eba368e94";
```
Comment (thiagokokada) [inline pkgs/applications/audio/pamixer/default.nix:10]: https://github.com/cdemoulins/pamixer/commit/4ea2594cb8c605dccd00a381ba19680eba368e94

File: pkgs/applications/audio/pamixer/default.nix [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
+    runHook preInstall
+
     install -Dm755 pamixer -t $out/bin
+
+    runHook postInstall
+  '';
+
+  postInstall = ''
+    gzip -c pamixer.1 > pamixer.1.gz
+    installManPage pamixer.1.gz
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/pamixer/default.nix:28]:
```suggestion
    installManPage pamixer.1
```
Dont automatically.

Comment (thiagokokada) [reply pkgs/applications/audio/pamixer/default.nix]: Nice.

Comment (thiagokokada) [reply pkgs/applications/audio/pamixer/default.nix]: Fixed.

File: pkgs/applications/audio/pamixer/default.nix [hunk 3]
```diff
@@ -1,20 +1,30 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, boost, libpulseaudio }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, boost, libpulseaudio, installShellFiles }:
 
 stdenv.mkDerivation rec {
   pname = "pamixer";
-  version = "1.4";
+  version = "unstable-2021-03-29";
```
Comment (figsoda) [inline pkgs/applications/audio/pamixer/default.nix:5]: currently having a version starting with unstable messes up `builtins.parseDrvName` and  `nix run`
my recommendation is to either set `meta.mainProgram` (what i would probably do) or use some other style of version, which there is an ongoing discussion about normalization: https://github.com/NixOS/rfcs/pull/107

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/pamixer/default.nix (hunk 3 above)
Comment (thiagokokada) [reply pkgs/applications/audio/pamixer/default.nix:5]: Done.

**Review by figsoda — APPROVED** [review]

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 139961` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>nwg-panel</li>
    <li>pamixer</li>
  </ul>
</details>

**Pull request merged**